frontend/src/modules/Metadata_Forms/components/MetadataFormFields.js

```diff
@@ -661,11 +721,16 @@ export const MetadataFormFields = (props) => {
               <Autocomplete
                 disablePortal
                 options={versionOptions.map((option) => {
-                  return { label: 'version ' + option, value: option };
+                  return {
+                    label: 'version ' + option.version,
+                    value: option.version,
+                    attached: option.attached_forms
+                  };
                 })}
                 value={'version ' + currentVersion}
                 onChange={async (event, value) => {
                   setCurrentVersion(value ? value.value : versionOptions[0]);
+                  setAttached(value ? value.attached : 0);
                   await fetchItems(value ? value.value : versionOptions[0]);
```

Comment:
these defaults need to be `versionOptions[0].version` or `null` and associated functions need to be able to handle null

Otherwise get an error when X out the selected Version in the Fields Tab